components/progressbar/index.css

```diff
@@ -64,99 +99,139 @@ governing permissions and limitations under the License.
   justify-content: space-between;
   align-items: center;
-  inline-size: var(--spectrum-progressbar-width);
+  font-size: var(--mod-progressbar-font-size, var(--spectrum-progressbar-font-size));
   vertical-align: top;
-}
 
-.spectrum-ProgressBar-track {
-  /* Visually apply border radius to child elements */
-  overflow: hidden;
-  inline-size: 100%;
-  block-size: var(--spectrum-progressbar-height);
-  border-radius: var(--spectrum-progressbar-border-radius);
-  z-index: 1; /* to fix a weird webkit bug with rounded corners and masking */
-}
+  inline-size: var(--mod-progressbar-size-default, var(--spectrum-progressbar-size-default));
+  max-inline-size: var(--mod-progressbar-max-size, var(--spectrum-progressbar-max-size));
+  min-inline-size: var(--mod-progressbar-min-size, var(--spectrum-progressbar-min-size));
 
-.spectrum-ProgressBar-fill {
-  border: none;
-  block-size: var(--spectrum-progressbar-height);
+  /* Label and Percentage */
+  .spectrum-ProgressBar-label,
+  .spectrum-ProgressBar-percentage {
+    text-align: start;
+    margin-block-start: var(--mod-progressbar-spacing-top-to-text, var(--spectrum-progressbar-spacing-top-to-text));
+    margin-block-end: var(--mod-progressbar-spacing-label-to-progressbar, var(--spectrum-progressbar-spacing-label-to-progressbar));
 
-  transition: width 1s;
-}
+    line-height: var(--mod-progressbar-line-height, var(--spectrum-progressbar-line-height));
 
-.spectrum-ProgressBar-label,
-.spectrum-ProgressBar-percentage {
-  text-align: start;
-  margin-block-end: var(--spectrum-progressbar-value-gap-y);
-}
+    &:lang(ja),
+    &:lang(zh),
+    &:lang(ko) {
+      line-height: var(--spectrum-progressbar-line-height-cjk, var(--mod-progressbar-line-height, var(--spectrum-progressbar-line-height)));
+    }
 
-.spectrum-ProgressBar-label {
-  flex: 1 1 0%;
-}
+    color: var(--mod-progressbar-text-color, var(--spectrum-progressbar-text-color));
+  }
 
-.spectrum-ProgressBar-percentage {
-  align-self: flex-start;
-  margin-inline-start: var(--spectrum-fieldlabel-side-padding-right);
-}
+  /* Label */
+  .spectrum-ProgressBar-label {
+    flex: 1 1 0%;
+  }
 
-.spectrum-ProgressBar--sideLabel {
-  display: inline-flex;
-  flex-flow: row;
-  justify-content: space-between;
-  inline-size: auto;
+  /* Percentage */
+  .spectrum-ProgressBar-percentage {
+    align-self: flex-start;
+    margin-inline-start: var(--mod-progressbar-spacing-label-to-text, var(--spectrum-progressbar-spacing-label-to-text));
+  }
 
+  /* Track */
   .spectrum-ProgressBar-track {
-    min-width: var(--spectrum-progressbar-width);
-    flex: 1 1 var(--spectrum-progressbar-width);
+    /* Visually apply border radius to child elements */
+    overflow: hidden;
+    inline-size: 100%;
+    block-size: var(--mod-progressbar-thickness, var(--spectrum-progressbar-thickness));
+    border-radius: var(--spectrum-progressbar-corner-radius);
+    z-index: 1; /* to fix a weird webkit bug with rounded corners and masking */
```

Comment:
hmm, did this bug get introduced with the core-tokens migration or was it already present before?

Reply:
It was already present. I'll see if I can still reproduce it.